device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ModuleClient.java

```diff
@@ -170,7 +167,7 @@ public ModuleClient(String connectionString, IotHubClientProtocol protocol, Stri
      * See <a href="https://github.com/Azure/azure-iot-sdk-java/blob/master/device/iot-device-samples/send-event-x509/pom.xml">this pom.xml</a> for an example of how to do this.
      */
     @Deprecated
-    public ModuleClient(String connectionString, IotHubClientProtocol protocol, SSLContext sslContext) throws ModuleClientException, URISyntaxException
```

Comment:
Same comment for these ModuleClientExceptions. These would be breaking changes to remove